dsc/src/subcommand.rs

```diff
@@ -436,18 +436,18 @@ pub fn resource(subcommand: &ResourceSubCommand, stdin: &Option<String>) {
                         if !found { continue; }
                     }
 
-                    methods = vec!["get".to_string()];
-                    if manifest.set.is_some() { methods.push("set".to_string()); }
-                    if manifest.test.is_some() { methods.push("test".to_string()); }
-                    if manifest.export.is_some() { methods.push("export".to_string()); }
+                    methods = vec!["g".to_string()];
```

Comment:
Each `to_string()` will result in an allocation.  I wonder if we can preallocate one string `----` and replace the characters instead to reduce allocations